healthlocker_service_script.md

```diff
@@ -0,0 +1,107 @@
+The first step is to open your terminal and make sure that you are in the root
+of the healthlocker Repo. Once here run the following...
+
+```
+cat healthlocker.service
+```
+
+This will print the contents of the file into your terminal. Highlight the
+contents and copy them to your clipboard with `cmd/control + c`
+
+The next step is to `ssh` onto the server with
+```
+ssh root@'ip_address'
+```
+
+Once on the server create a file called `'app_name'.service` in the
+`/lib/systemd/system/` directory. For example
+
+```
+touch /lib/systemd/system/healthlocker.service
+```
+
+Now that the file is created you will need to paste the code you have copied
+to your clipboard into the file. To do this you will need to...
+
+- vim `/lib/systemd/system/'app_name'.service`
```

Comment:
vim would be typed wouldn't it? so the whole line can be in ``

Reply:
👍 Thanks. Will update now